sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -270,6 +270,12 @@
       <artifactId>spring-core</artifactId>
       <version>5.2.10.RELEASE</version> <!-- {x-version-update;org.springframework:spring-core;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-webmvc</artifactId>
+      <version>5.2.10.RELEASE</version> <!-- {x-version-update;org.springframework:spring-webmvc;external_dependency} -->
+      <optional>true</optional>
+    </dependency>
```

Comment:
We need add this dependency in azure-spring-boot-starter-active-directory. (without `optional`).